analysis_options.yaml

```diff
@@ -3,3 +3,11 @@
 # BSD-style license that can be found in the LICENSE file.
 
 include: package:lints/recommended.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  exclude:
+    - example/lib/main.dart # Ignore placeholder example file.
+    - lib/src/generated_bindings.dart # Ignore generated files.
```

Comment:
Use `preamble` to add `// ignore_for_file: lint1, lint2, lint3...` at the top of files generated by ffigen.

Reply:
Got it. :)